pkg/load/agents/configAgent.go

```diff
@@ -17,8 +18,9 @@ import (
 // ConfigAgent is an interface that can load configs from disk into
 // memory and retrieve them when provided with a config.Info.
 type ConfigAgent interface {
-	GetConfig(metadata api.Metadata) (api.ReleaseBuildConfiguration, error)
-	GetAll() load.FilenameToConfig
```

Comment:
We had no users

Reply:
https://github.com/openshift/ci-tools/blob/master/pkg/webreg/webreg.go#L1851

Reply:
How does this compile then?

Reply:
my controller also uses this. Why does this PR compile?

Reply:
Weird, GoLand did not tell me there were any callers ....